src/platforms/index.mdx

```diff
@@ -13,6 +13,17 @@ To report to Sentry you’ll need to use a language-specific SDK. The Sentry tea
 
 If your use-case is very specific, or not covered by Sentry, you'll find documentation on how to report events on your own using our [_SDK API_](https://develop.sentry.dev/sdk/overview/).
 
+<Note>
+
+Sentry SDKs are not officially supported nor recommended for use in third-party SDKs and libraries due to technical limitations:
+
+- Sentry SDKs are not designed to be used in third-party SDKs and libraries.
```

Comment:
```suggestion
- Sentry SDKs are not designed to be used in plugins, packages, and libraries (for example payment SDKs or widgets to be embedded on websites)
```

@marandaneto and I clarified that "third-party SDKs" here meant SDKs completely unrelated to Sentry or error monitoring. Since I find "third-party SDKs" a bit ambiguous in this context, I vote we rephrase this a bit.